docs/miscellaneous.rst

```diff
@@ -64,7 +64,7 @@ The position of ``data[4][9].b`` is at ``keccak256(uint256(9) . keccak256(uint25
 Layout in Memory
 ****************
 
-Solidity reserves four 32 byte slots:
+Solidity reserves four 32-byte slots, with specific byte ranges (inclusive of endpoints) being used as follows:
```

Comment:
This reads as if the slots would specify byte ranges. Perhaps it should be written as "three special memory areas", one 64 byte range, `0x00 - 0x3f`: ..., one 32 byte range, etc.?

Reply:
>This reads as if the slots would specify byte ranges.
The slots do specify byte ranges, in the lines just after this - or am I misreading something here?

I think your suggestion is still an improvement, because it might not be intuitively clear that 0x00-0x3f (inclusive of endpoints) is decimal 64 bytes.

Reply:
I meant this can be read as if the memory contents inside these slots are interpreted as byte ranges.

Reply:
Oh I'm sorry, I just plainly misread the text. Yes, you are of course right.